test/fakes/fake_just_audio.dart

```diff
@@ -73,6 +73,14 @@ final icyMetadataMessage = IcyMetadataMessage(
   ),
 );
 
+extension SaveCompleter<T> on Completer<T> {
```

Comment:
you can remove the name to make it non-public like this `extension on Completer<T>`

Reply:
Nice, didn't know that was possible.